README.md

```diff
@@ -8,6 +8,47 @@ The `CRUD` module allows to perform CRUD operations on the cluster.
 It also provides the `crud-storage` role for
 [Tarantool Cartridge](https://github.com/tarantool/cartridge).
 
+# Supported Tarantool versions
+
+CRUD depends on Tarantool features like key_def, merger, UUID modules and JSON
+paths support. We do our best to provide uniform support for all supported
+Tarantool version with help of compatibility layer, runtime feature detection
+etc. but anyway some Tarantool versions has limited support in CRUD. Please
+consult with a list below to choose a Tarantool version compatible with CRUD
+module before use:
+
+- 1.10: since 1.10.7-91-g9ee14eca5 with tuple-merger<sup>1</sup> and
+  tuple-keydef<sup>2</sup> modules, without UUID keys comparison and without
+  Jsonpath indexes
```

Comment:
I don't see information about minimal tarantool version I can use with crud. It generally works without keydef and merger (built-in or external), correct? So, say, I can use it on 1.10.3 if I don't want JSON path indexes, UUID, decimal and sorting by a collation different than 'unicode' or 'unicode_ci'. Correct?

After PR #198 this rule splits to two:

* For all functions except min/max, <..as stated above..>.
* For min / max there is additional requirement <...>.

The compatibility guarantees were changed and I want to know how exactly. Any formal unambiguous form to present this information is okay.